jetstream/cli.py

```diff
@@ -219,54 +296,70 @@ def rerun(project_id, dataset_id, experiment_slug, config_file):
 
         config = spec.resolve(experiment)
 
-        # delete all tables previously created when this experiment was analysed
-        client = BigQueryClient(project_id, dataset_id)
-        normalized_slug = bq_normalize_name(experiment.normandy_slug)
-        analysis_periods = "|".join([p.value for p in AnalysisPeriod])
-        table_name_re = f"^(statistics_)?{normalized_slug}_({analysis_periods})_.*$"
-        client.delete_tables_matching_regex(table_name_re)
-
         for date in inclusive_date_range(experiment.start_date, end_date):
             logger.info(f"*** {date}")
             Analysis(project_id, dataset_id, config).run(date)
+
+        _update_tables_last_modified(project_id, dataset_id, experiment_slug)
```

Comment:
Instead of deleting all tables previously created when this experiment was analysed, we'll update the `last_updated` label of all tables matching the experiment slug. This feels a lot safer.